src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md

```diff
@@ -0,0 +1,11 @@
+---
+title: 'New OTel JVMs page' 
+summary: 'Compare JVM performance for OTel services' 
```

Comment:
Change title to 'Compare JVM performance metrics for OTel services on a new JVM page'

Reply:
@daynaslord, this new change seems to conflict with the previous change where the word "New" is used in the summary. Can we just stick with your previous suggestion?

Reply:
Sure, we can just change the title to 'Compare JVM performance metrics for OTel services'